features/step_definitions/brief_response_steps.rb

```diff
@@ -41,6 +41,19 @@
   }
 end
 
+Given 'that supplier has filled in their application but not submitted it' do
+  @brief_response = create_brief_response(@service['lotSlug'], @brief_id, @supplier['id'])
```

Comment:
Might be nicer if the log slug was taken from from the brief rather than the services the supplier offers but either way works